tests/integration-tests-utils/src/test/java/org/apache/bookkeeper/tests/MavenClassLoaderTest.java

```diff
@@ -39,4 +39,9 @@ public void testNoZooKeeperInterference() throws Exception {
                                    org.apache.zookeeper.KeeperException.NoNodeException.class));
     }
 
+    @Test
+    public void testLoadSpecifyingMavenRepo() throws Exception {
```

Comment:
Honestly I don't like very much to depend on remote resources.
The best would be for me to start an http server, with wiremo and then use it

Reply:
What's the issue with remote resources for a test that'll only normally run in CI? If github is down, the CI job will fail much earlier.

Reply:
I am not sure what is the purpose of this test? It seems that we are testing the functionality that provides by maven resolver. didn't see much value to do this though.

Reply:
What about dropping the testcase  ?

Reply:
@eolivelli actually I misunderstood your original point. Yes, unit tests shouldn't have remote resource dependencies. I thought you meant we shouldn't use this in the integration tests.
@sijie I wrote the test case before the code to validate that what I assumed the library would do is actually what the library did.

I think the test has value, but given the hassle of having to boot a http server and mock up a maven repo, just to test a test utility, I'm just going to remove the test.